src/auth-service/models/User.js

```diff
@@ -312,6 +292,13 @@ UserSchema.statics = {
           foreignField: "_id",
           as: "role",
         })
+        .unwind("$role")
+        .lookup({
+          from: "permissions",
```

Comment:
picking out user information for each role_permission. @BenjaminSsempala @Codebmk